java-core/src/main/java/ai/certifai/junit5/ex1/BasicMethodAssertion.java

```diff
@@ -1,12 +1,12 @@
-package solution;
+package ai.certifai.junit5.ex1;
 
 /**
  * Demo of Junit5 Test (Class)
```

Comment:
You can write some introduction and where do you put the test file, because beginner might dont know what is the purpose of this file.